server/src/game/role/mortician.rs

```diff
@@ -89,14 +78,24 @@ impl RoleStateImpl for Janitor {
     fn get_won_game(self, game: &Game, actor_ref: PlayerReference) -> bool {
         crate::game::role::common_role::get_won_game(game, actor_ref)
     }
-    fn on_phase_start(self, game: &mut Game, actor_ref: PlayerReference, _phase: PhaseType){
-        actor_ref.set_role_state(game, RoleState::Janitor(Janitor { cleans_remaining: self.cleans_remaining, cleaned_ref: None }));
+    fn on_phase_start(self, _game: &mut Game, _actor_ref: PlayerReference, _phase: PhaseType){
     }
     fn on_role_creation(self, _game: &mut Game, _actor_ref: PlayerReference){
-        
     }
     fn on_any_death(self, _game: &mut Game, _actor_ref: PlayerReference, _dead_player_ref: PlayerReference){
     }
+    fn on_grave_added(self, game: &mut Game, actor_ref: PlayerReference, grave_ref: GraveReference){
+        if actor_ref.alive(game) && self.cremated_players.contains(&grave_ref.deref(game).player) {
+            let grave = grave_ref.deref_mut(game);
+            grave.role = GraveRole::Cremated;
+            grave.will = "".to_owned();
+            let grave = grave.clone();
```

Comment:
You shouldn't need to clone here

Reply:
I tried to remove it and got borrow checker errors. Double borrow of game on the same line